smt/surrogate_models/surrogate_model.py

```diff
@@ -384,15 +385,17 @@ def predict_variances(self, x):
 
     def predict_variance_derivatives(self, x):
         """
-        Give the derivation of the variance of the kriging model (for one input)
+        Predict the derivation of the variance at a point
+
         Parameters:
         -----------
-        - x: array_like
-        Input
+        x : np.ndarray
+            Input value for the prediction point.
+
         Returns:
         --------
-        - derived_variance: array_like
-        The jacobian of the variance of the kriging model
+        derived_variance: np.ndarray
+            The jacobian of the variance
         """
         check_support(self, "derivatives")
         check_support(self, "variances")
```

Comment:
Much better thanks! Then you can replace the two lines above by <code>check_support(self, "variance_derivatives")</code>

Reply:
Should be enough to delete them since check_support(...) is called in _predict_variance_derivatives?

Reply:
I am going to add it to match the other methods.